src/operation/operation_model_interface.jl

```diff
@@ -297,23 +297,23 @@ function _list_names(model::OperationModel, container_type)
 end
 
 function read_dual(model::OperationModel, key::ConstraintKey)
```

Comment:
Are these functions meant to be used for the export? or by the user?

Reply:
Anything that takes a key should be internal only. Why do you ask?

Reply:
Just to see what you were thinking about the use of the functions.